Marlin/Configuration.h

```diff
@@ -2475,7 +2485,7 @@
  * just remove some extraneous menu items to recover space.
  */
 //#define NO_LCD_MENUS
-//#define SLIM_LCD_MENUS
+//#define SLIM_LCD_MENUS //  SLIM_LCD_MENUS will disable 50 mm moves and above
```

Comment:
`SLIM_LCD_MENUS` disables a lot more than 50/100mm moves and already has a comment above saying extraneous menu items will be removed.